packages/tldraw/src/lib/Tldraw.tsx

```diff
@@ -118,6 +122,10 @@ export function Tldraw(props: TldrawProps) {
 		[_tools]
 	)
 
+	if (embeds) {
+		EmbedShapeUtil.setEmbedDefinitions(embeds)
```

Comment:
This is updating global state, which we should avoid. Otherwise config might leak between tldraw instances.

I think a better way to do this would be to wait until after the Editor has been initialized, then find the instance of the EmbedShapeUtil by `editor.getShapeUtil('embed')` and calling `setEmbedDefinitions` on that, making it an instance method instead of a static method.

Would that work?